dockerversion/version.go

```diff
@@ -0,0 +1,21 @@
+package dockerversion
+
+import (
+	"golang.org/x/net/context"
+)
+
+// APIVersionKey is the client's requested API version.
+const APIVersionKey = "api-version"
```

Comment:
Please follow the best practices here. This should be a local type:

```
type apiVersionKey struct{}
```

This keeps storage requirements small and access local to this package.